hello-world/Dockerfile

```diff
@@ -13,13 +14,20 @@ LABEL name="Hello World" \
       summary="The Stackable image for the Stackable Hello World Operator" \
       description="This image is deployed by the Stackable Hello World Operator."
 
-RUN microdnf update && \
-    microdnf clean all && \
-    rm -rf /var/cache/yum
+RUN <<EOF
+microdnf update
+microdnf clean all
+rm -rf /var/cache/yum
```

Comment:
It looks like some images skip this step. Should be be done on all, or it is only needed here?

Nothing to change here, just wanted to ask.

Reply:
This should be done everywhere and I thought I checked that they did (at least the ones this PR is touching) but I'll check again, thanks!

Reply:
I checked. All the ones I touch have it.

Reply:
I did a quick check based on counting lines, and see there is a difference:

```
❯ rg --color=always 'microdnf update' | wc -l
42

❯ rg --color=always 'microdnf clean' | wc -l
38

❯ rg --color=always '/var/cache/yum' | wc -l
37
```